media-libs/intel-mediasdk/intel-mediasdk-20.2.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
```

Comment:
I pulled these into my local overlay and had to add `cmake-utils` to the inherits to prevent this:
```>>> Configuring source in /var/tmp/portage/media-libs/intel-mediasdk-20.2.0/work/MediaSDK-intel-mediasdk-20.2.0 ...
/var/tmp/portage/media-libs/intel-mediasdk-20.2.0/temp/environment: line 2006: cmake-utils_src_configure: command not found
>>> Source configured.```
(Confusingly, the final error message from portage for this was that the build.ninja wasn't found.)